src/bluesky/callbacks/olog.py

```diff
@@ -173,5 +173,5 @@ def start(self, doc):
             pformat(doc),
         )
         olog_status = self.client.log(document_content, logbooks=self.logbook)
-        logger.debug("client.log returned %s", olog_status)
+        logger.debug(f"client.log returned {olog_status}")
```

Comment:
```suggestion
        logger.debug("client.log returned %s", olog_status)
```

Calling it this way defers paying the cost of turning `olog_status` into a string until we actually need it.  Even if it is cheap here, it is best-practice do logging calls this way.

Reply:
the change was with an autoformatting tool, likely ruff. maybe we should add a comment that we do for performance reasons the change into using %s and add a noqa comment

Reply:
done